docs/api/apiv3/paths/storage_files.yml

```diff
@@ -6,12 +6,12 @@ get:
   tags:
     - File links
   description: |-
-    Gets a collection of file data from a storage. 
+    Gets a collection of files from a storage.
     
-    If no `root` context was given, the result is the content of the document root. With `root` context given, the
-    result contains the collections of file data from without the given root file id.
+    If no `root` context is given, the result is the content of the document root. With `root` context given, the
```

Comment:
As you know I already got fooled by the word `root` here. A file tree has only one root. What you probably want to say what the root for this request shall be. For me calling it "parent" would make much more sense. It actually is a "node" or "directory" in the file tree (which has just one root). And if this endpoint also accepts files (leaves of the tree) as parameters then I'd call it something like "storageItem". However, please see my comment below as I am still confused about the general idea of this API endpoint.